cryptostore/config.py

```diff
@@ -45,26 +53,76 @@ def __contains__(self, item):
 
 class DynamicConfig(Config):
     def __init__(self, file_name=None, reload_interval=10, callback=None):
-        if file_name is None:
-            file_name = os.path.join(os.getcwd(), 'config.yaml')
-        if not os.path.isfile(file_name):
-            raise FileNotFoundError(f"Config file {file_name} not found")
+        # Normal boto3 credentialing methods are used (see https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html)
+
+        self.s3_object_uri = os.environ.get('S3_CONFIG_FILE_URI') #returns None if env var not present
+        self.s3_region     = os.environ.get('S3_REGION') #returns None if env var not present
+        LOG.debug(f'self.s3_object_uri is: {self.s3_object_uri}')
+
+        if self.s3_object_uri is (None or ''):
```

Comment:
can this just be `if not self.s3_object_uri` ?

Reply:
Yes, I'll make that change, it was an edge case left over from debugging that I haven't been able to reproduce. Also, I removed a duplicate call to _load that was a duplicate of the calls from with the if / else conditions.